cmd/ddltest/ddl_serial_test.go

```diff
@@ -118,6 +118,11 @@ func createDDLSuite(t *testing.T) (s *ddlSuite) {
 	require.NoError(t, err)
 	ddl.RunWorker = false
 	session.ResetStoreForWithTiKVTest(s.store)
+	s.dom.Close()
```

Comment:
Is this introduced by #28423?

Reply:
Yes or no ... The goroutine leaks already happens before #28423 . But it did fails after it because after it gorotuine leaks check was enabled